FirebaseAppCheck/Sources/Public/FirebaseAppCheck/FIRAppCheck.h

```diff
@@ -74,6 +74,36 @@ NS_SWIFT_NAME(AppCheck)
 /// set explicitly, the value will be persisted and used as a default value on next app launches.
 @property(nonatomic, assign) BOOL isTokenAutoRefreshEnabled;
 
+/// Requests Firebase app check token. This method should *only* be used if you need to authorize
```

Comment:
Do the changes in this file need an API review?

Reply:
I have a feeling we probably should. I haven't seen a precedent for it but I'd also like to add an APITests file for `FirebaseAppCheckInterop`.

Reply:
Note: The lines in this file were deleted in https://github.com/firebase/firebase-ios-sdk/pull/11660/files#diff-eac252484339eb3e41865e21a844b340aaa72faedd3b703c50d3af1646575de2L55-L84 because the APIs were copied into `FIRAppCheckProtocol`. I added them back because the types now differ between `FIRAppCheckProtocol` and `FIRAppCheck`. Adding them back silenced the warning:
> Conflicting parameter types in implementation of 'tokenForcingRefresh:completion:': 'void (^ _Nonnull __strong)(id<FIRAppCheckTokenProtocol>  _Nullable __strong, NSError * _Nullable __strong)' vs 'void (^ _Nonnull __strong)(FIRAppCheckToken * _Nullable __strong, NSError * _Nullable __strong)'

Reply:
SGTM. Interesting though, I would have expected the generic to show up `(id _Nullable __strong, NSError * _Nullable __strong)` to be `(id<token protocol> _Nullable __strong, NSError * _Nullable __strong)`